docs/modules/ROOT/pages/config/quarkus-zookeeper.adoc

```diff
@@ -262,15 +262,4 @@ Environment variable: `+++QUARKUS_ZOOKEEPER_SESSION_CAN_BE_READ_ONLY+++`
 --|boolean 
 |`false`
 
-
-a| [[quarkus-zookeeper_quarkus.zookeeper.health.enabled]]`link:#quarkus-zookeeper_quarkus.zookeeper.health.enabled[quarkus.zookeeper.health.enabled]`
-
-[.description]
---
-Whether to enable health checks.
-
-Environment variable: `+++QUARKUS_ZOOKEEPER_HEALTH_ENABLED+++`
---|boolean 
-|`true`
-
-|===
```

Comment:
This is not the right fix as this file is automatically generated.

The problem is that this configuration property is present in both the runtime config and the build time config and is thus being generated twice. That's what needs fixing.

/cc @morphy76

Reply:
https://github.com/quarkiverse/quarkus-zookeeper-client/issues/71

Reply:
@morphy76  Cool, thanks!